tools/test-case-generators/format-request-map.json

```diff
@@ -171,7 +171,10 @@
         }
       }
     },
-    "overrides": {}
+    "overrides": {},
+    "supported_arches": [
+      "x86_64"
```

Comment:
Out of curiosity, what is the reason for not testing the `fedroa-iot-commit` image type with kernel customization on `aarch64`?

Reply:
On fedora35 at least, there doesn't seem to be packages for `kernel-debug` or `kernel-rt` in aarch64:
```bash
$> dnf list kernel-rt
Last metadata expiration check: 0:19:52 ago on Wed 16 Mar 2022 11:48:01 AM EDT.
Error: No matching Packages to list
```
```bash
$> dnf list kernel-debug
Last metadata expiration check: 0:19:34 ago on Wed 16 Mar 2022 11:48:01 AM EDT.
Error: No matching Packages to list
```
There's only `kernel`:
```bash
$> dnf list kernel
Last metadata expiration check: 0:21:26 ago on Wed 16 Mar 2022 11:48:01 AM EDT.
Installed Packages
kernel.aarch64 
```

Reply:
Thanks for the explanation!